web/themes/joinup/joinup_theme.theme

```diff
@@ -992,3 +1018,18 @@ function joinup_theme_preprocess_table__file_formatter_table(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add theme suggestions directly to the comment body field.
+ */
+function joinup_theme_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (strpos($form['#id'], 'views-exposed-form-search-page-') !== FALSE) {
+    $form['#info']['filter-search_api_fulltext']['label'] = '<span class="search-bar__icon icon icon--search"></span>';
```

Comment:
@kalinchernev Please add your browser and OS X name and version information to can reproduce issue with positioning because on latest Chrome ver 59 and OS X 10.12 it works correctly.